src/chapters/classes-part2/equals-shortcut.rst

```diff
@@ -0,0 +1,72 @@
+Visual Studio Generator Shortcut
+================================
+
+Instead of cutting, pasting, and refactoring old code to ensure that you create a well-structured ``GetHashCode()`` method whenever you define your own ``Equals()`` method, you can use Visual Studio’s code generation tool!
+Just right-click in your class file on the class name and follow these prompts.
+
+Let’s use a Course class to demonstrate:
+
+.. sourcecode:: csharp
+   :linenos:
+
+   public class Course
+    {
+            public string Topic { get; set; }
+            public Teacher Instructor { get; set; }
+            public List<Student> enrolledStudents { get; set; }
+    }
+
+#. In Visual Studio, right-click on the class name and select *Quick Fix* from the menu.
+
+   .. figure:: figures/select-quick-fix.png
+      :alt: Screenshot showing the Quick Fix option at the top of the menu when one right-clicks on the class name.
+
+#. Select the *Generate Equals() and GetHashCode()* option from the resulting menu:
+
+   .. figure:: figures/generate-equals.png
```

Comment:
i like the screen shots here but is it too annoying to ask that we show the same thing in the demo app we ask the students to use? csharp-web-dev-lsn3and4classes?